Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/silicon_base.yml

```
@@ -179,7 +178,7 @@
       doAfterDelay: 32
     - type: Bloodstream
       damageBleedModifiers: BloodlossIPC
-      bloodReagent: Water
+      bloodReagent: MotorOil
```

Their "blood" is meant to be coolant, they overheat if they bleed
```suggestion
      bloodReagent: Water
```